front/src/modules/trainschedule/components/ManageTrainSchedule/PacedTrainSettings/AddedOccurences.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+
+import { Clock, Calendar, Trash, Dot } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import { addException, deleteException } from 'reducers/osrdconf/operationalStudiesConf';
+import { getExceptions } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { useAppDispatch } from 'store';
+
+const AddedOccurences = () => {
+  const [date, setDate] = useState('');
+  const [time, setTime] = useState('');
+  const exceptions = useSelector(getExceptions);
+  const { t } = useTranslation('operational-studies', { keyPrefix: 'manageTrainSchedule' });
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    const now = new Date();
+    setDate(now.toISOString().slice(0, 10));
+    setTime(now.toTimeString().slice(0, 5));
```

Comment:
I'm not a huge fan of the string offset shenanigans. Maybe we could add a new function similar to `formatLocalDateTime()`/`formatLocalDate()` but for time strings?

Reply:
done